doc/contributing/primordials.md

```diff
@@ -124,7 +124,7 @@ performance of code in Node.js.
 * `SafePromisePrototypeFinally`: use `try {} finally {}` block instead.
 * `ReflectConstruct`: Also affects `Reflect.construct`.
   `ReflectConstruct` creates new types of classes inside functions.
```

Comment:
I know I am nitpicking myself.

Maybe should be "at runtime" and not "inside functions". 

Sorry.